src/devices/Bmxx80/samples/Bme280.sample.cs

```diff
@@ -46,7 +47,7 @@ static void Main(string[] args)
                     i2CBmpe80.TryReadTemperature(out var tempValue);
                     Console.WriteLine($"Temperature: {tempValue.Celsius} C");
                     i2CBmpe80.TryReadPressure(out var preValue);
-                    Console.WriteLine($"Pressure: {preValue} Pa");
+                    Console.WriteLine($"Pressure: {preValue.Pascal} Pa");
```

Comment:
optional: change these to show hPa instead since it's more useful

Reply:
I tried to not change as much from the logic of the samples. I noticed they need to be more aligned.

Reply:
Done